packages/desktop-client/src/components/BankSyncStatus.tsx

```diff
@@ -58,8 +56,15 @@ export function BankSyncStatus() {
                   iconStyle={{ color: theme.pillTextSelected }}
                 />
                 <Text style={{ marginLeft: 5 }}>
-                  Syncing... {accountsSyncingCount} account
-                  {accountsSyncingCount > 1 && 's'} remaining
+                  <Trans
+                    i18nKey="syncingAccounts"
+                    count={count}
+                    values={{ count }}
+                  >
+                    <Trans count={count}>
+                      Syncing... {{ count }} account remaining
+                    </Trans>
+                  </Trans>
```

Comment:
Same here please, and the pluralised form is preferred by default

```suggestion
                  <Trans count={accountsSyncingCount}>
                    Syncing... {{ count }} accounts remaining
                  </Trans>
```